boa3/builtin/interop/storage/storagecontext.py

```diff
@@ -0,0 +1,4 @@
+class StorageContext:
+    def __init__(self):
+        self.id: int = 0
```

Comment:
As StorageContext is an InteropInterface in Neo, we can't access the properties, so this class shouldn't have any property.